sources/kube.go

```diff
@@ -247,7 +247,7 @@ func newKubeSource() (*KubeSource, error) {
 		Insecure: true,
 	})
 
-	glog.Infof("Using Kubernetes client %+v\n", kubeClient)
+	glog.Infof("Using Kubernetes client with master %q and version 'v1beta1'\n", *argMaster)
```

Comment:
nit: Can we constify "v1beta1" here, below, and in the struct above so they all come from one variable?